simulators/ethereum/eest/execute/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Builds and runs the EEST (execution-spec-tests) execute hive simulator
+FROM ghcr.io/astral-sh/uv:python3.10-bookworm-slim
+
+## Default fixtures/git-ref
+ARG fixtures=stable@latest
+ENV FIXTURES=${fixtures}
+ARG branch=main
+ENV GIT_REF=${branch} 
+
+## Clone and install EEST
+RUN apt-get update && apt-get install -y git
+
+# Allow the user to specify a branch or commit to checkout
+RUN git init execution-spec-tests && \
+    cd execution-spec-tests && \
+    git remote add origin https://github.com/ethereum/execution-spec-tests.git && \
+    git fetch --depth 1 origin $GIT_REF && \
+    git checkout FETCH_HEAD;
+
+WORKDIR /execution-spec-tests
+RUN uv sync
+
+# Cache the fixtures. This is done to avoid re-downloading the fixtures every time
+# the container starts.
+# If newer version of the fixtures is needed, the image needs to be rebuilt.
+# Use `--docker.nocache` flag to force rebuild.
+RUN uv run consume cache --input "$FIXTURES"
```

Comment:
We don't need to cache anything here since we don't use the fixtures, but this raises an interesting question regarding how to pin `execute` to a release.